libethcore/SealEngine.h

```diff
@@ -47,6 +47,11 @@ class SealEngineFace
 public:
 	virtual ~SealEngineFace() {}
 
+    virtual std::tuple<h256, h256, h256> getWork(BlockHeader const& _bi)
+    {
+        return std::tuple<h256, h256, h256>(_bi.hash(WithoutSeal), h256(0), h256(0));
```

Comment:
I guess this hash calculation is not needed
```suggestion
        return std::tuple<h256, h256, h256>{};
```